docs/getting-started-guides/index.md

```diff
@@ -25,13 +25,14 @@ a Kubernetes cluster from scratch.
 
 ### Local-machine Solutions
 
-Local-machine solutions create a single cluster with one or more Kubernetes nodes on a single
-physical machine.  Setup is completely automated and doesn't require a cloud provider account.
-But their size and availability is limited to that of a single machine.
+[Minikube](/docs/getting-started-guides/minikube) (recommended starting point) creates a
+single node kubernetes cluster locally for purposes of development and testing. Minikube
```

Comment:
I agree. Minikube is good for anything the API can do. But not the right solution if you need to test networking, storage, etc. For that I believe local VMs via vagrant is still the right solution.